cmd/snap/cmd_list.go

```diff
@@ -124,6 +131,60 @@ func listSnaps(names []string, all bool) error {
 	w := tabWriter()
 	defer w.Flush()
 
+	switch format {
+	case "":
+		return outputSnapsDefault(w, snaps)
+	default:
+		return outputSnapsWithFormat(Stdout, snaps, format)
+	}
+}
+
+func clientSnapFields() []string {
+	v := reflect.ValueOf(client.Snap{})
+	n := v.Type().NumField()
+	fields := make([]string, n)
+	for i := 0; i < n; i++ {
+		fields[i] = v.Type().Field(i).Name
+	}
+
+	sort.Strings(fields)
```

Comment:
I don't think we want to sort the fields: they're defined in a more or less sensible order, with related things placed next to each other. That's a nice thing to have.